hooks/tk-multi-publish2/basic/collector.py

```diff
@@ -372,6 +383,36 @@ def collect_sg_writenodes(self, parent_item):
 
             self.logger.info("Collected file: %s" % (publish_path,))
 
+            if review_submission_app is None:
+                error_msg = "Review Submission App is not available. Will not add review movie for export"
+                self.logger.error(error_msg)
+                raise Exception(error_msg)
+            else:
+                review_submission_item = item.create_item("nuke.reviewsubmission", "Movie File", display_name + " Review Submission")
```

Comment:
Do we need to create a separate item for review submission? I was hoping we could get away with simply adding a plugin to attach to existing `*.sequence` item types. So in the item tree you'd see 2 plugins attached to the image sequence item: **Publish to Shotgun** and **Submit for Review**.